pkg/resource/pod_resource_manger.go

```diff
@@ -64,7 +62,7 @@ func NewPodResourceManager(client clientset.Interface, nodeName string, podInfor
 		runtimeClient: runtimeClient,
 		runtimeConn:   runtimeConn,
 		stateChann:    stateChann,
-		collectors:    collectors,
+		Manager:       cadvisorManager,
```

Comment:
change to private object？

Reply:
+1

Reply:
This is a  anonymous member variable, the interface is called Manager.